docs_app/content/tutorial/8-subjects.md

```diff
@@ -0,0 +1,304 @@
+# 8. Subjects
+
+## TLDR:
+
+- Both an [observer](GL) and an [observable](GL). Inheriting from [`Observable`](API).
+- The primary use case is [multicasting](GL).
+- The next use case is eventing, such as dealing with single-function event handlers in frameworks like React and Angular.
+- A less common use case is a [reactive loop](#subjects-reactive-loops---a-tertiary-use-case.)
+- Should not be used as substitute for creating an [`Observable`](API).
+- Subjects are not reuseable. Once `error` or `complete` is called, the subject will no longer emit.
+
+## "Subjects" Are From The Observer Pattern
+
+The subject type, as it exists in RxJS, is really derivative of the subject you'd find the the [Observer Pattern](WIKIPEDIA). The observer pattern is as follows: There are two types, the "subject" and the "observer". They have an API that looks, _generally_, like this:
+
+**Observer Pattern Types** (Prefixed with `OP_` because this is not RxJS)
+
+```ts
+interface OP_Observer<T> {
+    /** Handle a value notification */
+    notify(value: T): void;
+}
+
+interface OP_Subject<T> {
+    /** Register an observer to be notified by this subject */
+    addObserver(observer: OP_Observer<T>): void;
+    /** Unregister an observer so it stops being notified by this subject */
+    removeObserver(observer: OP_Observer<T>): void;
+    /** Notify all registered observers of a value */
+    notifyObservers(value: T): void;
+}
+```
+
+If you look at the API above, and you squint a little, you'll notice by renaming a few things you'll find some familiar-looking methods in these interfaces. For one, you could rename `notify` to `next` in `OP_Observer`, and you'll have a partial RxJS [`Observer`](API) type. Also, looking at `OP_Subject`, we know of a type that has a method that accepts an observer: [`subscribe`](API) on [`Observable`](API). So if we rename `addObserver` to `subscribe`, we're pretty close. But what about `removeObserver`? That's easy, we can get rid of it entirely, and instead return a [`Subscription`](API) object that, when [unsubscribed](GL), will remove the observer from the subject for us.
```

Comment:
Consider reordering the prepositional phrases: "If you look at the API above, squint a little, and rename a few things, you'll notice some familiar-looking methods in these interfaces."